org.eclipse.xtext.xbase.tests/src/org/eclipse/xtext/xbase/tests/compiler/CompilerTests2.java

```diff
@@ -0,0 +1,2193 @@
+/**
+ * Copyright (c) 2012, 2020 itemis AG (http://www.itemis.eu) and others.
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.xtext.xbase.tests.compiler;
+
+import org.eclipse.xtext.util.JavaVersion;
+import org.eclipse.xtext.xbase.compiler.GeneratorConfig;
+import org.eclipse.xtext.xbase.compiler.output.FakeTreeAppendable;
+import org.eclipse.xtext.xbase.lib.ObjectExtensions;
+import org.eclipse.xtext.xbase.lib.Procedures.Procedure1;
+import org.junit.Test;
+
+public class CompilerTests2 extends AbstractOutputComparingCompilerTests {
+
+	@Override
+	protected FakeTreeAppendable createAppendable() {
+		return new FakeTreeAppendable();
+	}
+
+	/**
+	 * https://bugs.eclipse.org/bugs/show_bug.cgi?id=424279
+	 */
+	@Test
+	public void testBug_424279() throws Exception {
+		compilesTo(
+				"{\n" +
```

Comment:
Do you have a script for these changes?

Reply:
not bug regexes in eclipse find dialog ;)